base/db/R/insert.format.vars.R

```diff
@@ -126,13 +126,13 @@ insert.format.vars <- function(con, format_name, mimetype_id, notes = NULL, head
       u1 <- formats_variables[1,"unit"]
       u2 <- dplyr::tbl(con, "variables") %>% dplyr::select(.data$id, units) %>% dplyr::filter(.data$id %in% !!formats_variables[[1, "variable_id"]]) %>% dplyr::pull(.data$units)
 
-      if(!udunits2::ud.is.parseable(u1)){
+      if(!units::as_units(u1)){
```

Comment:
I think I suggested this earlier but didn't think through the wrappers needed: If the units aren't parseable this will throw an error rather than return FALSE, so we'll need to wrap it in a tryCatch.

```suggestion
      u1_recognized <- tryCatch(units::as_units(u1), error = function(e) FALSE)
      if(!u1_recognized){
```

(Apparently {units} does define a `ud_is_parseable()` but doesn't export it, because that would have been too easy...)